jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/MicroprofileHealthUtil.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.common.util;
+
+import org.eclipse.jkube.kit.common.Dependency;
+import org.eclipse.jkube.kit.common.JavaProject;
+
+import static org.eclipse.jkube.kit.common.util.SemanticVersionUtil.isVersionAtLeast;
+
+public class MicroprofileHealthUtil {
+  private static final String MICROPROFILE_HEALTH_DEPENDENCY = "microprofile-health";
+  private static final String MICROPROFILE_DEPENDENCY = "microprofile";
+  private static final String MICROPROFILE_GROUP = "org.eclipse.microprofile";
+  private static final int MICROPROFILE_HEALTH_SUPPORTED_VERSION_MAJOR = 3;
+  private static final int MICROPROFILE_HEALTH_SUPPORTED_VERSION_MINOR = 1;
+  public static final String DEFAULT_READINESS_PATH = "/health/ready";
+  public static final String DEFAULT_LIVENESS_PATH = "/health/live";
+  public static final String DEFAULT_STARTUP_PATH = "/health/started";
+
+  private MicroprofileHealthUtil() { }
+
+  public static boolean hasMicroProfileHealthDependency(JavaProject javaProject) {
+    return JKubeProjectUtil.hasDependency(javaProject, MICROPROFILE_GROUP , MICROPROFILE_HEALTH_DEPENDENCY);
+  }
+
+  public static boolean hasMicroProfileDependency(JavaProject javaProject) {
+    return JKubeProjectUtil.hasDependency(javaProject, MICROPROFILE_GROUP , MICROPROFILE_DEPENDENCY);
+  }
+
+  public static boolean isStartupEndpointSupported(JavaProject javaProject) {
+    String microProfileHealthVersion = "";
+    if (hasMicroProfileDependency(javaProject)) {
```

Comment:
Maybe this part could be improve I see 2 blocks of code that are doing the same thing with `MICROPROFILE_DEPENDENCY` and `MICROPROFILE_HEALTH_DEPENDENCY`